lib/processors/jsdoc/lib/transformApiJson.cjs

```diff
@@ -5,6 +5,8 @@
  * Licensed under the Apache License, Version 2.0 - see LICENSE.txt.
  */
 
+/* eslint-disable */
```

Comment:
This is a bit unfortunate and you might not like it. But I would prefer to keep the files in sync as much as possible. 

One possible way out would be to make this an external config in openui5 (using a local .eslintrc.json for the lib or for the jsdoc folder) and remove the comment from this file here.

Reply:
Fine for me

Reply:
Fine with me too